javascript/javascript1/week1/Housey - pricey.js

```diff
@@ -21,9 +21,9 @@ const costPayingByPeter = 2500000;
  //Julia house price
```

Comment:
you can use array format for volume and for the price of the house as well like: 
const volumeInMeters = [widthOfHouse[0]*heightOfHouse[0]*depthOfHouse[0], widthOfHouse[1]*heightOfHouse[1]*depthOfHouse[1]];
compare both the values using index value of an array.